rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -319,9 +319,90 @@ paths:
                     items:
                       $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/General/ForbiddenEntity.yml#/ForbiddenError"
 
-# ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-# AUTOMATED RECORDING REQUESTS
-# ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+  # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+  /user/{user_id}/recordings_quota:
+    get:
+      description: |
+        * **Available since:** 2020-1 (BPLAT-12326)
+      tags:
+        - Recordings
+      summary: Fetch used and total available recordings quota in seconds
+      operationId: userRecordingsQuota
+      parameters:
+        - in: path
+          name: user_id
+          required: true
+          description: The identity of the user to which the quota belongs to.
+          schema:
+            type: string
+      responses:
+        200:
+          description: "Successful response"
+          content:
+            application/json:
+              schema:
+                type: "object"
+                required:
+                  - data
+                properties:
+                  data:
+                    type: object
```

Comment:
Please create an entity for this schema and reference it so that it can be reused